plenum/config.py

```diff
@@ -59,7 +60,7 @@
 clientBootStrategy = ClientBootStrategy.PoolTxn
 
 hashStore = {
-    "type": HS_FILE
+    "type": HS_LEVELDB
```

Comment:
could you please use rocksdb here

Reply:
No, thanks)

Reply:
Seriously speaking, I've made temporary roll back to leveldb as rocksdb building procedure is not ready yet.